src/components/elements/headers.tsx

```diff
@@ -11,7 +11,7 @@ export const H1 = ({children, className, ...props}: Props) => {
       {...props}
       className={twMerge(
         "rs-mb-5 font-medium",
-        className?.match(/(type-|text-)/) ? className : ["type-3 xl:text-[4.1rem]", className]
+        className?.match(/type-\btext-\d+\b/g) ? className : ["type-3 xl:text-[4.1rem]", className]
```

Comment:
This should now check to see if there's `type-` and `text-#`. Previously it was just looking for `type-` and `text-` since I didn't account for our `text-[color]` and `text-[alignment]` utility classes 💀